camlibs/quicktake1x0/qtkt-decoder.c

```diff
@@ -0,0 +1,122 @@
+/* qtkt-decoder.c
+ *
+   Copyright 1997-2018 by Dave Coffin, dcoffin a cybercom o net
+ * Copyright 2023, Colin Leroy-Mira <colin@colino.net>
+ *
+ * QTKT decoder heavily inspired from dcraw.c.
+ *
+ * This file is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
```

Comment:
libgphoto2 is LGPL. That's a problem.

Reply:
I was wondering about this too, but thought it should be okay given the number of files mentioning GPL in camlibs:

```
grep -lr "GNU General Public License" camlibs/
```

If it isn't, I can drop ppm conversion and only implement .qtk export.

Reply:
also note that dcraw is not GPL, expect the "restricted" section which relate only to foveon.
```
No license is required to download and use dcraw.c.  However,
   to lawfully redistribute dcraw, you must either (a) offer, at
   no extra charge, full source code* for all executable files
   containing RESTRICTED functions, (b) distribute this code under
   the GPL Version 2 or later, (c) remove all RESTRICTED functions,
   re-implement them, or copy them from an earlier, unrestricted
   Revision of dcraw.c, or (d) purchase a license from the author.
```

Reply:
Yes, I've read that paragraph a few times and I *do think* it means I can relicense those two files as LGPL if you'd prefer. However I'm not a lawyer, so I preferred to play it safe after seeing quite a lot of GPL-licensed files in camlibs. 
If you prefer me to relicense as LGPL and also think I'm in the clear doing it, I'll gladly do it!

Reply:
not sure about these other camlibs files :-/ it's either a mistake or the pandora box has been open.

Reply:
> Yes, I've read that paragraph a few times and I _do think_ it means I can relicense those two files as LGPL if you'd prefer. However I'm not a lawyer, so I preferred to play it safe after seeing quite a lot of GPL-licensed files in camlibs. If you prefer me to relicense as LGPL and also think I'm in the clear doing it, I'll gladly do it!

This is "(c) remove all RESTRICTED functions, re-implement them, or copy them from an earlier, unrestricted Revision of dcraw.c": the restricted code is not part of this. The restricted code starts at line 3046 and end at 3748 in the original dcraw 1.478.

Reply:
Yes, and it's an "either".
I'll update the headers tomorrow morning